Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -1018,6 +1018,29 @@ class NewTabPageSetting: Setting {
     }
 }
 
+class SiriPageSetting: Setting {
```

Comment:
I would consider making this entire class @/available iOS 12. It will prevent you having to check within and will prevent it from ever being created when unnecessary. Just a bit safer.